### PR TITLE
Bit 120 fisher validator

### PR DESCRIPTION
**Pull request by isabella618033**
1. fisher info
2. more wandb logging
3. separate learning rate for chain_weight (peer_weight)
4. template miner sync with block time

### CODE REVIEW

File: miners/text/template_validator.py [hunk 1]
```diff
@@ -36,30 +38,51 @@
...
     bittensor.dataloader.add_args( parser )
+    
+    # ---- Loads config_file and updates defaults
+    config_file_path = vars(parser.parse_known_args()[0])['miner.config']
+    if config_file_path:
+        config_file_path = os.path.expanduser(config_file_path)
+        try:
+            with open(config_file_path) as f:
+                params_config = yaml.safe_load(f)
+                print('Config File Detected at {} updating defaults'.format(config_file_path))
```
Comment (shibshib) [inline miners/text/template_validator.py]: Should this be print or a logger.info?

File: miners/text/template_validator.py [hunk 2]
```diff
@@ -95,31 +119,48 @@ def main( config ):
...
+
+        def query ( self, inputs ):
+
+            # ---- Get active peers and their weights ---- 
+            active_uids = torch.where(metagraph.active > 0)[0]
+            active_peer_weights = self.peer_weights[active_uids]
+
+            # ---- Topk Weights ---- (TODO: check if the gaussians are enough disrupt the chain weights)
+            real_topk = min( config.nucleus.topk, metagraph.n.item(), len(active_uids))
+            noise = torch.normal( 0, torch.std(active_peer_weights).item()+0.0000001, size=( active_peer_weights.size())).to( config.miner.device )
```
Comment (shibshib) [inline miners/text/template_validator.py]: Any reason for 0.0000001? If not, we should turn this into a constant

**Comment by shibshib** [comment]
Conflictsssss! Not sure if this is because of @unconst's changes or @isabella618033 :D
